src/_scss/pages/award/shared/awardAmountsSection/_charts.scss

```diff
@@ -5,6 +5,8 @@ $new-potential: #AAC6E2;
 $file-c-outlay: #6E338E;
 $file-c-obligation: #B699C6;
 
+$old-obligated: #4773aa;
```

Comment:
`$color-cool-blue-light`
Would also eventually like to make the COVID purple colors into variables to ensure they are consistent across pages